### DISCUSSION TITLE
Is it possible to use Ollama which is running on other local machines?

### DISCUSSION THREAD

**Discussion by kanoto** [TheR1D/shell_gpt#512] [Q&A] [answered]
Hello, I have two machines under the same local network: one hosts Ollama and the other does not. I'm curious whether it's possible to set up SGPT on the second machine to utilize the Ollama hosted on the first machine. Is that possible?

**Comment by quantdev** [comment, marked as answer]
I was able to define the API_BASE_URL to use the remote Ollama server.  ( Please replace <<REMOTE_HOST>> to your host IP or hostname.

API_BASE_URL=http://<<REMOTE_HOST>>:11434

Also I have to allow the ollama server to connect from a remote machine.

**Reply by kanoto** [reply]
It works, thank you!

**Comment by gwpl** [comment]
Seems like new issue arised: https://github.com/TheR1D/shell_gpt/discussions/503
